docs/conflict_resolution.md

```diff
@@ -0,0 +1,81 @@
+## Conflict Resolution 
```

Comment:
Add info that library works with the offix-sever that is node.js based and conflict protocol descibed on offix.dev. 

Explain that sample app implements version support, however, devs can use whatever marker they want. We can link to offix.dev documentation. 

I will bring documentation back to the offix later.

Reply:
@lavanyagaur22